src/components/PetCard/PetCard.css

```diff
@@ -68,4 +68,75 @@
 
 .detail-link-icon {
   height: 70px;
+}
+
+@media screen and (max-width: 1364px) {
+  .pet-card {
+    width: 280px
+  }
+}
+
+@media screen and (max-width: 1287px) {
+  .pet-card {
+    width: 250px
+  }
+}
+
+@media screen and (max-width: 1175px) {
+  .pet-card {
+    width: 200px;
+  }
+}
+
+@media screen and (max-width: 989px) {
+  .pet-card {
+    width: 170px;
+    height: 180px
+  }
+  .pet-icon {
```

Comment:
This one also changed the image size in PetDetails. Maybe we can go in and look at it